lib/travis/build/script/r.rb

```diff
@@ -39,7 +44,7 @@ def export
           sh.export 'TRAVIS_R_VERSION', 'release', echo: false
         end
 
-        def setup
+        def configure
```

Comment:
I don't actually think it is possible for users to override `setup` or `configure` (see https://travis-ci.org/jimhester/travis-test/builds/103650273#L86 for an example of trying to do so). Neither is one of the [normal build steps](https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle).